src/actions/chat.js

```diff
@@ -1,16 +1,48 @@
 
+import uuidV1 from 'uuid/v1';
```

Comment:
@bonham000  About this... I'm already using `shortid` for flash messages, we could prob shed a dependency and stick with one or the other. Any reason to go with this over `shortid`? 

shortids are shorter I guess so theoretically could repeat sooner, but somehow I doubt that will be an issue.